internal/connect/hwinfo.go

```diff
@@ -269,3 +276,40 @@ func readValues2map(b []byte) map[string]string {
 	}
 	return m
 }
+
+// Returns the parsed value for the given file. The implementation has been
+// split from `systemMemory` so testing it is easier, but bear in mind that
+// these two are coupled.
+func parseMeminfo(file io.Reader) int {
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		fields := strings.Fields(scanner.Text())
+		if len(fields) < 2 || fields[0] != "MemTotal:" {
+			continue
+		}
+
+		val, err := strconv.Atoi(fields[1])
+		if err != nil {
+			Debug.Printf("could not obtain memory information for this system: %v", err)
+			return 0
+		}
+		return val / 1024
```

Comment:
Not sure why we are doing this. Maybe the scc api already expects MiB.

Reply:
Yes, this was agreed with @digitaltom to just for for MiB, since we will usually care about this in the factor of TiB, so sending mere KiB might be an overkill.